larray/core/session.py

```diff
@@ -369,46 +372,57 @@ def load(self, fname, names=None, engine='auto', display=False, **kwargs):
         --------
         In one module:
 
+        >>> # scalars
+        >>> i, s = 5, 'string'
         >>> # axes
-        >>> a, b = Axis("a=a0..a2"), Axis("b=b0..b2")    # doctest: +SKIP
+        >>> a, b = Axis("a=a0..a2"), Axis("b=b0..b2")
```

Comment:
there is a reason all these doctests were skipped! This is because they depend on pytables which is an optional dependency!